.gitignore

```diff
@@ -15,7 +15,13 @@ build
 # Generated files
 .docusaurus
 .cache-loader
-.yarn
+
+# Yarn
+.yarn/cache
+.yarn/unplugged
+.yarn/build-state.yml
+.yarn/install-state.gz
+.pnp.js
```

Comment:
better use ignores from this guide https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored